pkg/render/manager.go

```diff
@@ -1011,3 +995,21 @@ func managerClusterWideDefaultView() *v3.UISettings {
 		},
 	}
 }
+
+func DefaultSGLayerTigeraNamespaces() map[string]bool {
```

Comment:
Maybe you can add a note that a map is used for deduplication only and that bools have no meaning (since you are not filtering out namespaces that are false).

Reply:
Done